tests/check_ncb64.c

```diff
@@ -144,6 +163,7 @@ Suite *config_suite(void)
         tcase_add_test(t, check_nc_b64_enc_n_file_foob);
         tcase_add_test(t, check_nc_b64_enc_n_file_foobar);
         tcase_add_test(t, check_nc_b64_enc_n_file_long_text);
+	tcase_add_test(t, check_nc_b64_enc_n_filehandler_foobar);
```

Comment:
The indentation looks off for a bunch of lines in this commit... According to the mode line, indentation should be spaces only, and the indentation level is 8 spaces.